pages/linux/lshw.md

```diff
@@ -0,0 +1,19 @@
+# lshw
+
+> List detailed information about hardware configurations as root user.
+
+- Launch the GUI:
+
+`sudo lshw -X`
+
+- List all hardwares with table format:
```

Comment:
I think _in tabular_ format sounds better.

Reply:
Yes! I've updated the description.